### ISSUE TITLE
Let attic take a remote path option for cases where attic cannot be installed in standard places (like rsync, git, etc) 

### ISSUE THREAD

**Issue by sergiocallegari** [jborg/attic#333]
Most programs that support remote operation via ssh requiring a server on the remote host, let one specify the path of the remote host.

For instance, `rsync` supports the `--rsync-path=PROGRAM`  to specify the rsync to run on remote machine.
Git has something similar for the full path of git-upload-pack on the remote host.

Conversely, in attic, the remote path seems to be hardwired to `attic`, which creates issues whenever attic cannot be installed in standard places.

**Comment by ThomasWaldmann** [comment]
could this be solved by adjusting the remote user's PATH?

**Comment by sergiocallegari** [comment]
No, unfortunately it cannot and this is why other software using a similar ssh based remote access have an option for the full path of the remote server.

Issue is that when ssh is passed a command, the command is executed on the remote host /instead/ of a login shell. This means that the command is executed in the system default environment, without the option to personalize the path or anything in `.bashrc`. And I think this is done for security reasons.

**Comment by ThomasWaldmann** [comment]
@sergiocallegari ok, understood! I'll assimilate this issue. :)
